src/main/cpp/t04_students.cpp

```diff
@@ -29,10 +29,37 @@
 
 #include "t04_students.h"
 #include <iostream>
+#include <algorithm>
+#include <map>
+#include <vector>
+#include <set>
+#include <string>
 
 using namespace std;
-
+bool sx(double i,double j){
+    return i>j;
+}
 int t04_students() {
-    
+     int n;
+    cin>>n;
+    string a,b,c;
+    double x,y,z;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'h' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4586321)